src/cascadia/ShellExtension/OpenTerminalHere.cpp

```diff
@@ -49,17 +49,17 @@ try
         siEx.StartupInfo.wShowWindow = SW_SHOWNORMAL;
 
         std::filesystem::path modulePath{ wil::GetModuleFileNameW<std::wstring>(wil::GetModuleInstanceHandle()) };
+        modulePath.replace_filename(runElevated ? ElevateShimExe : WindowsTerminalExe);

```

Comment:
@DHowett Previously we invoked `wt.exe` here (and have done so since the shell extension was first added). Do you have an idea if we should continue to do that? @zadjii-msft also can't quite remember why it's been built like that. 🤔

Also, is there any advantage to continue using the `IsPackaged()` path in `GetWtExePath()`? Seems like it doesn't actually do anything that the non-packaged code already does.

Reply:
Thanks for fixing the command line arg problem.

I don't know why we specifically launched `wt.exe`... it may have just been a ye-olden-days choice. I hope so.

Reply:
> Seems like it doesn't actually do anything that the non-packaged code already does.

Oh, I see. This is interesting.
I think it **USED TO** use wt.exe because that was the app execution alias. It would look up the app execution alias, specifically, out of `%LOCALAPPDATA%\Microsoft\WindowsApps\<<PackageFamilyName>>`.

I wonder if we did this to try to avoid the "bad permissions on WindowsApps" bug... bah. I can't recall.

Reply:
No, that code still exists.

There's a difference there. `GetWtExePath` points to the _app execution alias_ file if we're packaged. It points to on-disk `wt.exe` if we're not packaged.